source/docs/articles/local/drush-on-pantheon.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How Drush Works on Pantheon
+description: Learn how to create a policy file and sitealias alter hook.
```

Comment:
Suggest alternate title, maybe: **Altering Drush Behavior with a Policy File**

Suggest alternate description: "Learn how to use hook implementations within a policy file to modify the default behavior or version of Drush."